### DISCUSSION TITLE
We are not fairing well against the competition

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6122] [Announcements]
[[21st century Amphibia, collected vs observed](https://discourse.gbif.org/t/21st-century-amphibia-collected-vs-observed/3903)](https://discourse.gbif.org/t/21st-century-amphibia-collected-vs-observed/3903)

The short version is our data is awful and is such a small part of the picture it hardly matters anyway. What are we doing to increase data quality and ensure that our data (and therefore specimens) are seen as valuable?

**Comment by ewommack** [comment]
I don't think that is a fair comparison at all. A lot of the things the author complained about are really relevant to science. We are being honest in that we cannot identify some amphibians in a museum collection, but putting that down means that the author labels the record as unusable. The fact that the animal is present in the collection for further identification should compound the value of it for a taxonomist.

**Comment by DerekSikes** [comment]
I also wouldn't describe human observation data as 'competition' - there's plenty of room for both types of data. Even if observations becomes 95% of the data, museums are still relevant if someone wants DNA or morphology to study.

**Comment by dustymc** [comment]
>e still relevant if someone wants DNA or morphology to study.

Or evidence of any kind. (And "we" don't help that cause by generally refusing to cite things in some way that helps separate the 'IDK, maybe made it up" from the "there's the item, feel free to examine it, yay museums.")

Much of the rest is just not following GBIF's occasionally strange and always cryptic worldview eg https://www.youtube.com/watch?v=3uBcq1x7P34

**Reply by Jegelewicz** [reply]
That video was awesome! Now do "species"....

**Comment by Jegelewicz** [comment]
FWIW - I know there are "reasons" this comparison has flaws, but I don't think we should fell OK saying "we are better because we have stuff" and just keep rolling. We DO need to pay attention to data quality, otherwise our data will get lost in the masses of "quality" observational data.

**Reply by ewommack** [reply]
But I don't think its a quality issue in our data. I think it is how they are designing and thinking about how to use data.

**Reply by dustymc** [reply]
>pay attention to data quality

Absolutely, but it's hard to see the actual problems when we're lost in the weeds of GBIF complaining about things that look somewhere between arbitrary (continents) and just wrong (refusing to use our identifiers) from here.
